bookkeeper-server/src/main/java/org/apache/bookkeeper/http/ListUnderReplicatedLedgerService.java

```diff
@@ -112,7 +110,7 @@ public HttpServiceResponse handle(HttpServiceRequest request) throws Exception {
                     return response;
                 }
             } catch (Exception e) {
-                e.printStackTrace();
+                LOG.error("Meet Exception: ", e);
```

Comment:
typo. Met Exception. 
In any case, it would be better to describe what you were doing when the exception occurred.
"Exception occurred while listing underreplicated ledgers", e

Reply:
Thanks, will change it.